instaclustr/structs.go

```diff
@@ -71,16 +71,17 @@ type RackAllocation struct {
 }
 
 type CreateRequest struct {
-	ClusterName           string          `json:"clusterName"`
-	Bundles               []Bundle        `json:"bundles"`
-	Provider              ClusterProvider `json:"provider"`
-	SlaTier               string          `json:"slaTier"`
-	NodeSize              string          `json:"nodeSize"`
-	DataCentre            string          `json:"dataCentre"`
-	ClusterNetwork        string          `json:"clusterNetwork"`
-	PrivateNetworkCluster string          `json:"privateNetworkCluster"`
-	PCICompliantCluster   string          `json:"pciCompliantCluster"`
-	RackAllocation        *RackAllocation `json:"rackAllocation,omitempty"`
+	ClusterName           string                    `json:"clusterName"`
+	Bundles               []Bundle                  `json:"bundles"`
+	Provider              ClusterProvider           `json:"provider"`
+	SlaTier               string                    `json:"slaTier,omitempty"`
+	NodeSize              string                    `json:"nodeSize"`
+	DataCentre            string                    `json:"dataCentre,omitempty"`
+	DataCentres           []DataCentreCreateRequest `json:"dataCentres,omitempty"`
```

Comment:
I feel the naming will be confusing, because it seems we need to define multiple `data_centres` in the plan, rather than a list named `data_centres`. Ideally we should just update `DataCentre` to be the slice, but that would bring some backward compatibility issue. Can we make sure it's clear on the usage?

Also, is it possible to use the already-existing `DataCentre` struct? It will allow specifying more things as per ChrisW's comment?

Reply:
I've chat to Stephen offline and agreed to having the separate DataCentreRequest struct - we just need to add more fields